src/components/modals/BaseModalERC20Import.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <base-modal :loading="loading" :error="error" title="Import Token Contract" id="modalERC20Import">
+        <template #modal-body v-if="!loading">
+            <!-- <base-form-select-network @selected="chainId = $event" /> -->
+            <b-form-group>
+                <base-drop-down-select-quick-swap-toke-list @selected="onSelectToken" />
+            </b-form-group>
+        </template>
+        <template #btn-primary>
+            <b-button
+                :disabled="loading || erc20TokenAddress == ''"
+                class="rounded-pill"
+                @click="submit()"
+                variant="primary"
+                block
+            >
+                Import ERC20 Token
+            </b-button>
+        </template>
+    </base-modal>
+</template>
+
+<script lang="ts">
+import { ChainId } from '@/types/enums/ChainId';
+import { TERC20 } from '@/types/erc20';
+import { Component, Vue } from 'vue-property-decorator';
+import { mapGetters } from 'vuex';
+// import BaseFormSelectNetwork from '../form-select/BaseFormSelectNetwork.vue';
+import BaseDropDownSelectQuickSwapTokeList from '../dropdowns/BaseDropdownSelectQuickSwapTokeList.vue';
+import BaseModal from './BaseModal.vue';
+
+@Component({
+    components: {
+        BaseModal,
+        // BaseFormSelectNetwork,
+        BaseDropDownSelectQuickSwapTokeList,
+    },
+    computed: mapGetters({}),
+})
+export default class ModalERC20Import extends Vue {
+    loading = false;
+    error = '';
+    chainId: ChainId = ChainId.PolygonMumbai;
```

Comment:
@valeriagrazzini This should be ChainId.Polygon since all the tokens from the list are on polygon mainnet (and not testnet -> Mumbai)